app/Http/Controllers/SitemapController.php

```diff
@@ -56,7 +56,7 @@ public function developer(){
         return Sitemap::render();
     }
 
-    public function board(){
+    public function board() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->